admin_guide/quota.adoc

```diff
@@ -520,11 +520,6 @@ ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 [[limited-resources-quota]]
 == Require Explicit Quota to Consume a Resource
 
-[NOTE]
-====
-This feature is tech preview and subject to change in future releases.
-====
-
 If a resource is not managed by quota, a user has no restriction
```

Comment:
This is still a tech preview feature.  It’s built on aloha configuration flags upstream